### PR TITLE
Table - expand/collapse all demo

### PR DESCRIPTION
**Pull request by jschuler**
**What**: Closes https://github.com/patternfly/patternfly-react/issues/4943

https://patternfly-react-pr-5030.surge.sh/components/table/react-demos#expandcollapse-all

### CODE REVIEW

**Comment by jschuler** [comment]
cc @mcoker @mcarrano 
https://patternfly-react-pr-5030.surge.sh/components/table/react-demos#expandcollapse-all

File: packages/react-table/src/demos/Table.md [hunk 1]
```diff
@@ -243,8 +260,185 @@ class BulkSelectTableDemo extends React.Component {
...
+  toggleSelectAll(checked) {
+    this.onSelect(null, checked, -1);
+  }
+
+  render() {
+    const { columns, rows, expandCollapseToggle, isChecked } = this.state;
+
+    return (
+      <React.Fragment>
+        <Toolbar>
```
Comment (mcoker) [inline packages/react-table/src/demos/Table.md:398]: If you add `<ToolbarContent>` and use a `<ToolbarGroup>`, the alignment is improved. 

Screen Shot 2020-10-15 at 11 03 18 AM

@mcarrano wdyt, should it look like this?

**Comment by mcoker** [comment]
@mcarrano should the icon in the expand all button in the toolbar rotate like it does when you click on it in the table?

**Review by dlabrecq — APPROVED** [review]
LGTM. Although, I have a couple questions.

1. Does the expand all apply to all table rows or only the visible ones? 
2. Should the expanded state be maintained during pagination?

**Comment by mcarrano** [comment]
Looks great @jschuler .

> @mcarrano should the icon in the expand all button in the toolbar rotate like it does when you click on it in the table?

Yes, I think it should @mcoker . @mmenestr is that also what you expect? Also, see @dlabrecq 's questions. I had not thought about this but would be interested in your input.

**Comment by mmenestr** [comment]
@mcoker yes! I wrote out all the behavior expectations in[ this doc ](https://docs.google.com/document/d/1zEtmy0z4DfTDwZNei-Wgd-04ox8H3NQxA8z18-109QU/edit#heading=h.kljurod7nnaq), hopefully that helps.

and re Dan's questions, I would say yes to both? It would apply to all table rows, and when you paginate through, it would show the rows expanded still - until the point where you decide to collapse all. That's what makers sense to me anyway. Open to thoughts!

**Review by mcoker — APPROVED** [review]

**Comment by jschuler** [comment]
there will be a followup PR to address the animation of the toggle

**Pull request merged**